src/components/weather-page/resources/weather_page_summary_icon_cloud.svg

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
As discussed offline, I think this would be a little nicer as a component

```jsx
export function CloudIcon {
    return (
        <svg viewBox="0 0 60 60">
            stuff...
        </svg>
    );
}
```